xla/service/gpu/runtime/nccl_all_reduce_thunk.h

```diff
@@ -77,7 +77,8 @@ class NcclAllReduceStartThunk : public NcclAllReduceReduceScatterThunkBase {
  protected:
   absl::Status RunNcclCollective(const ExecuteParams& params,
                                  se::Stream& stream,
-                                 NcclApi::NcclCommHandle comm) override;
+                                 NcclApi::NcclCommHandle comm,
```

Comment:
Maybe change everything to wrapper type everywhere? You have header with that struct included anyway

Reply:
updated